mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -1,11 +1,30 @@
 AUI.add(
 	'todo',
 	function(A) {
+		new A.CharCounter(
+			{
+
+				input: '.input-item',
+
+				counter: '.counter',
+
+				maxLength: 40
+		});
```

Comment:
Initialize element in 'initializer', this is a place for constants declaration. Also indentation and spacing is wrong, see any function with object as argument.